postgresql-11.5/src/bin/psql/mainloopbteq.c

```diff
@@ -0,0 +1,630 @@
+/*
+ * psql - the PostgreSQL interactive terminal
+ *
+ * Copyright (c) 2000-2018, PostgreSQL Global Development Group
+ *
+ * src/bin/psql/mainloop.c
+ */
```

Comment:
update comments. this relates to all duplicated files.

Reply:
Done